WebExample/babel.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (api) {
     console.log('Starting Web example without Babel plugin.');
   }
   return {
-    presets: ['babel-preset-expo'],
+    presets: [['../preset', {disableBabelPlugin}], 'babel-preset-expo'],
```

Comment:
The idea here was that the user doesn't need to setup Babel plugin, so now we should disable Babel preset.